docs/content/patterns/review-and-submit/index.mdx

```diff
@@ -0,0 +1,305 @@
+
+The ‘Edit’ page will appear as a sub-step of the ‘Review and submit’ Progress indicator item.
+
+Once the changes are saved, the user will be returned to the ‘Review and submit’ page, where a success message will be displayed above the updated information in the summary list.
+
+### Review and submit sub-step
+
+```jsx live
+<Columns gap={{ xs: 1, md: 3 }}>
+	<Column columnSpan={{ xs: 12, md: 4 }}>
+		<ProgressIndicator
+			activePath="#review/page-1"
+			items={[
+				{ label: 'Page 1', status: 'done', href: '#page-1' },
+				{ label: 'Page 2', status: 'done', href: '#page-2' },
+				{
+					items: [
+						{
+							label: 'Page 1 sub step',
+							href: '#review/page-1',
+						},
+					],
+					label: 'Review and submit',
+					status: 'started',
+					href: '#review',
+				},
+			]}
+		/>
+	</Column>
+	<Column columnSpan={{ xs: 12, md: 8 }}>
+		<Stack gap={3}>
+			<DirectionLink direction="left" href="#">
+				Back
+			</DirectionLink>
+			<Stack gap={1.5}>
+				<H1>
+					<Text
+						color="muted"
+						display="block"
+						fontSize="sm"
+						fontWeight="bold"
+						lineHeight="heading"
+					>
+						Title of multi page form
+					</Text>
+					Page 1 title
+				</H1>
+				<Text as="p" color="muted" fontSize="md">
+					Check your name and contact details.
+				</Text>
+			</Stack>
+			<FormStack>
+				<TextInput hideOptionalLabel label="Term" value="Description" />
+				<TextInput hideOptionalLabel label="Label text" value="text" />
+				<TextInput hideOptionalLabel label="Label text" value="text" />
+			</FormStack>
+			<Divider />
+			<ButtonGroup>
+				<Button>Save and continue</Button>
+				<Button variant="secondary">Save and exit</Button>
+				<Button variant="tertiary">Cancel</Button>
+			</ButtonGroup>
+		</Stack>
+	</Column>
+</Columns>
+```
+
+### Editing a sub-page from ‘Review and submit’
+
+To edit information on the ‘Review and submit’ page, users can click the ‘Change’ link available under each summary list on the ‘Review and submit’ page.
+
+If the information they need to edit was originally provided via a sub-step of a form page, users will be taken to that sub-step on the ‘Review and submit’ page. From there, they will need into navigate to that form’s level 2 page to make the change.
+
+When editing information within a level 2 page, there will be no progress indicator visible. Instead, the user will be presented with a simple, single-page form. Users can navigate with the ‘Save and continue’ action at the bottom, or the back button at the top of the page.
+
+A user must save and continue on the level 2 form and the sub-step before those changes are saved and updated on the ‘Review and submit’ page.
+
+To avoid data loss, the [Warn before leaving pattern](/patterns/warn-before-leaving) should be used if the user elects to navigate back to the ‘Review and Submit’ page with any other method than ‘Save and continue’.
```

Comment:
```suggestion
To avoid data loss, the [Warn before leaving pattern](/patterns/warn-before-leaving) should be used if the user elects to navigate back to the ‘Review and submit’ page with any other method than ‘Save and continue’.
```